### ISSUE TITLE
Include SASS/SCSS file with distribution

### ISSUE THREAD

**Issue by dlewand691** [xdamman/selection-sharer#45]
Would love to have a .scss file included so I can import into my projects. Currently I copy the .css file out and change the extension to .scss. Having the .scss included in the distribution ensures that I'm using the latest version.

Thanks!

**Comment by shri3k** [comment]
I don't see any `sass` being included in this project so I don't understand why it should have `.scss` as an extension.

**Comment by dlewand691** [comment]
Exactly. There's no sass file but would be nice to have one so it can be customized with variables, etc.

**Comment by Dan503** [comment]
Did you know that you can import .css files into sass projects by just importing the file and leaving off the file extension?

So instead of this:
`````
@import 'path/to/file.css';
`````

Try this:
`````
@import 'path/to/file';
`````

It should magically work.
